### DISCUSSION TITLE
how to use adaptive activation functions

### DISCUSSION THREAD

**Discussion by hwangjinho94** [lululxvi/deepxde#1387] [Q&A]
Dear,

hello 
I am a student who is interested in PINN and my major is civil eng, 
I've read your Locally adaptive activation functions paper and I've been inspired a lot.

I tried to apply it to our engineering problem by referring to the link below.
"https://deepxde.readthedocs.io/en/latest/modules/deepxde.nn.html?highlight=adaptive%20active"

I tried to apply it to my code, but there was  an error occurred
I tried the same with the easiest code you released, but I ran into the same problem.

Please refer to the attached file. An easy example is attached.

What's the problem?
If you don't mind, could you provide me with a full code that applies LAAF?

Thank you.

[simple_example.pdf](https://github.com/lululxvi/deepxde/files/12066052/simple_example.pdf)

**Comment by lululxvi** [comment]
Check the examples at https://github.com/lu-group/deeponet-extrapolation

**Reply by ryabs07** [reply]
Is it possible to use LAAF without DeepOnet and within PyTorch?

**Reply by lululxvi** [reply]
Yes.

**Reply by JinglaiZheng** [reply]
Can you use LAAF on PyTorch now? Thank you very much @ryabs07

**Reply by ryabs07** [reply]
Hi,
I havent managed to do it yet. If you have any progress, please let me know :)
